<launch>

<node pkg="rosbag" type="play" name="player" args="-l $(find reggie_support)/bags/kinect.bag"/>

<node name="rviz" pkg="rviz" type="rviz" args="-d $(find reggie_support)/rviz/crop_cloud.rviz" />

<node name="crop_cloud_node" pkg="reggie_locator" type="crop_cloud" output="screen" >

  <!-- DEFAULT MAP
  <param name="x_min" type="double" value="-0.726849" />
  <param name="x_max" type="double" value="0.706064" />
  <param name="y_min" type="double" value="-0.543809" />
  <param name="y_max" type="double" value="0.267617" />
  <param name="z_min" type="double" value="1.014" />
  <param name="z_max" type="double" value="1.331" />
  -->

  <!-- YELLOW MAP
  <param name="x_min" type="double" value="0.387" />
  <param name="x_max" type="double" value="0.405" />
  <param name="y_min" type="double" value="0.145" />
  <param name="y_max" type="double" value="0.16" />
  <param name="z_min" type="double" value="0.9" />
  <param name="z_max" type="double" value="1.34" />
  -->

  <!-- GREEN MAP
  <param name="x_min" type="double" value="-0.4" />
  <param name="x_max" type="double" value="-0.38" />
  <param name="y_min" type="double" value="0.135" />
  <param name="y_max" type="double" value="0.15" />
  <param name="z_min" type="double" value="0.9" />
  <param name="z_max" type="double" value="1.34" />
  -->

  <!-- BLUE MAP
  -->
  <param name="x_min" type="double" value="0.399" />
  <param name="x_max" type="double" value="0.42" />
  <param name="y_min" type="double" value="-0.5" />
  <param name="y_max" type="double" value="-0.48" />
  <param name="z_min" type="double" value="1.014" />
  <param name="z_max" type="double" value="1.331" />
  <!--
  -->

</node>

</launch>
